packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -20,7 +20,7 @@ type TAppContent = {
     title: string;
 };
 
-type TPlatform = 'ctrader' | 'linux' | 'macos' | 'web' | 'windows';
+type TPlatform = 'ctrader_ios' | 'ctrader' | 'linux' | 'macos' | 'web' | 'windows';
```

Comment:
Refactor this constant file based on each platfrom (MT5, cTrader, derivX) and add all the details for mobile, desktop under one key